CRM/Group/Page/AJAX.php

```diff
@@ -50,7 +50,7 @@ public static function getGroupList() {
       CRM_Utils_JSON::output($groups);
     }
     else {
-      $params = CRM_Core_Page_AJAX::defaultSortAndPagerParams();
+      $params = array_merge($params, CRM_Core_Page_AJAX::defaultSortAndPagerParams());
```

Comment:
This does not appear to be the correct method, what paramaters are missing should be passed through as special params and escaped as per the methods in other AJAXs @mollux can you confirm that should be the approach here?